src/screens/home/hooks/useLauncherContext.tsx

```diff
@@ -95,7 +96,11 @@ export const LauncherContextProvider: React.FC<Props> = ({
             <ErrorParallelKonnectors
               currentRunningKonnector={launcherContext.value.konnector.slug}
               concurrentKonnector={concurrentKonnector}
-              onClose={(): void => setConcurrentKonnector(undefined)}
+              onClose={(): void => {
+                setConcurrentKonnector(undefined)
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+                launcherEvent.emit('launchResult', { cancel: true })
```

Comment:
I'm not sure to understand this one. How do we distinguish this `launchResult` from the one that would be emited by the "already running" connector?

Reply:
Actually, there is no distinction. The current account/konnector opened in harvest will receive this message. And since the current opened connector is the one which caused to open the "already running" connector, this is the one we want to cancel.